packages/brokeneck-react/src/components/Groups.js

```diff
@@ -112,41 +157,33 @@ export default function Groups() {
             </TableHead>
             <TableBody>
               {data?.groups.data.map(group => (
-                <TableRow key={group[groupFields.id]}>
+                <TableRow
+                  key={group[groupFields.id]}
+                  onClick={() =>
+                    history.push(`${match.url}/${group[groupFields.id]}`)
+                  }
```

Comment:
I see that you want to change this but not using links for navigation is not something I like much. the information about where clicking something navigates you to should live (also) in the html, and not just be in js. let's find a way  that preserves the existing behavior of using links